packages/desktop/components/drawers/network-config/drawers/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as NetworkCard } from './NetworkCard.svelte'
```

Comment:
For now we can put this in the `desktop/components` folder since it's not technically a drawer (unlikely to be used elsewhere, but still just for the sake of organization).